src/components/renderers/controls/datetime/datetime-renderer.spec.ts

```diff
@@ -16,7 +16,7 @@ describe('DateTimeTest', () => {
             }
         };
 
-    it("should be rendered",
+    xit("should be rendered",
```

Comment:
I guess theses tests are bootstrap specfic? Does it make sense to replace them then? If so, do we track this as an issue?